app/javascript/app/components/country/country-ndc-overview/country-ndc-overview-component.jsx

```diff
@@ -25,7 +25,8 @@ class CountryNdcOverview extends PureComponent {
         { type: 'info', onClick: handleInfoClick },
         {
           type: 'share',
-          shareUrl: `/embed/countries/${iso}/ndc-content-overview`
+          shareUrl: `/embed/countries/${iso}/ndc-content-overview`,
+          reverseDropdown: true
```

Comment:
When changing sections through the `anchorNav` the share menu (reverse)dropdown gets hidden under the `header` and the `anchorNav` forcing user to scroll the page to access the actions on the dropdown. I understand that the dropdown hides module content when open but maybe is preferable than having to scroll to click the desired share action?

Reply:
![image](https://user-images.githubusercontent.com/6906348/42212725-407081bc-7eb7-11e8-807a-b893e0331915.png)

Reply:
I just changed all the share dropdowns to open downwards right so we don't have this problem